docs/en/setup/examples/deploy/kubernetes/README.md

```diff
@@ -0,0 +1,28 @@
+# Kubernetes Deploy
+
+It could help you run the Satellite as a gateway in Kubernetes environment.
+
+## Install
+
+We recommend install the Satellite by `helm`, follow command below, it could start the latest release version of SkyWalking Backend, UI and Satellite.
+
+```shell
+export SKYWALKING_RELEASE_NAME=skywalking  # change the release name according to your scenario
+export SKYWALKING_RELEASE_NAMESPACE=default  # change the namespace to where you want to install SkyWalking
+export REPO=skywalking
+
+helm repo add ${REPO} https://apache.jfrog.io/artifactory/skywalking-helm                                
+helm install "${SKYWALKING_RELEASE_NAME}" ${REPO}/skywalking -n "${SKYWALKING_RELEASE_NAMESPACE}" \
+  --set oap.image.tag=8.8.1 \
+  --set oap.storageType=elasticsearch \
+  --set ui.image.tag=8.8.1 \
+  --set elasticsearch.imageTag=6.8.6 \
+  --set satellite.enabled=true \
+  --set satellite.image.tag=v0.3.0
```

Comment:
Is 0.3.0 correct?

Reply:
Update to the ready-to-release version, `v0.4.0`.